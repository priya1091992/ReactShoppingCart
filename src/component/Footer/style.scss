.footer {
  margin-top: 50px;
  background-color: lightGrey;
  padding: 10px;
}
