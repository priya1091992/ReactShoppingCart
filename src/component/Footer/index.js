import React, { Component } from 'react';
import './style.scss';

class Footer extends Component {
  render () {
    return (
      <div className="footer">
        {`Footer`}
      </div>
    )
  }
}

export default Footer
