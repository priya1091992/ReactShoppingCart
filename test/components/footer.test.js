import React from 'react';
import Footer from './../../src/component/Footer/index';
import renderer from 'react-test-renderer';

test('Footer component', () => {
  const component = renderer.create(
    <Footer />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
