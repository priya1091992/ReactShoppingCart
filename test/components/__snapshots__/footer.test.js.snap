// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component 1`] = `
<div
  className="footer"
>
  Footer
</div>
`;
